waterdata/views.py

```diff
@@ -53,6 +76,10 @@ def monitoring_location(site_no):
             'STATION_FIELDS_D'  : STATION_FIELDS_D
         }
         http_code = 200
+        # don't want to create more DOM elements if we don't have to
```

Comment:
Consider just using {% if json_ld %} in the template, then the if statement can be eliminated (see http://jinja.pocoo.org/docs/2.10/templates/#if)